_Week8/Group3/README.md

```diff
@@ -0,0 +1,74 @@
+
+## Motivation
+
+For this week's simulation assignment, we chose to illustrate the importance of intracluster correlation and the resulting design effect in clustered research designs.
+
+### ICC
+
+The intracluster correlation coefficient is a measure of how data point within clusters are related. The ICC is an important consideration when conducting power calculations and interpreting the results of data from a clustered study design. This is because similarity among the observations in clusters makes it more difficult to detect the true differences between study arms by reducing response variability. The ICC measures this similarity by comparing within cluster variance to between cluster variance.
+
+Note that accounting for similarities among clustered subjects usually results in power loss. Thus, with higher levels of intracluster correlation, more observations are required to obtain the same minimal detectable effect.
+
+A high icc ($\rho$) represents comparatively high "between clusters" variance.
+
+If $\rho=1$, then all responses within a cluster are identical.
+
+If $\rho=0$, then there is no correlation of responses within a cluster and the observations are functionally independent.
+
+### Design Effect
+
+During power calculations, the ICC and the average cluster size are used to calculate the design effect, which is included in the power calculation formula to adjust the required sample size for cluster sampling to make up for info lost from the clustered design. In essence, the design effect encapsulates the scalar increase in the MDE when a clustered design is adopted. A larger ICC, with the same average cluster size, produces a larger design effect, decreasing the minimal detectable effect.
+
+## simulate.do Description
+
+Our goal is to create simulated data across 200 clusters, which are schools in our case. In particular, we have defined an rclass program that uses the desired intracluster correlation coefficient ("ICC") as an input to the data generation process. Thus, we can input an expected ICC value and calculate the resulting design effect based on our simulated data. This is essential information for calculating our ultimate sample size.
+
+To use the program, type [schools, rho(VALUE)] where "VALUE" is the ICC you want to look at. Notably, this must be between 0 and 1.
+
+We then set the number of schools (clusters) to 200 and generate a way to refer to each of them.
+
+By generating n_students and expanding, we are randomizing the number of students per school and essentially replacing each school observation with "n_students" many copies.
+
+Then, we generate (student) ids to differentiate the observations in each cluster.
+
+### Intracluster correlation in the data
+
+Now, we are able to generate intracluster correlation in the data based on the desired rho. One of our assumptions for the sake of this simulation is that the total variance (variance between clusters + variance within clusters) = 1.
```

Comment:
This could be due to my lack of understanding of ICC but does varying the total variance have any implications? Might be good to mention if that's the case.